x/concentrated-liquidity/client/grpc/grpc_query.go

```diff
@@ -1,4 +1,5 @@
-package grpc 
+
```

Comment:
It probably doesn't matter but the package should be the first line, are we able to modify the template to do this for all of the below?

Reply:
Discussed offline and concluded that this is fine as long as no errors in the editor.

This stems from this line in the template:
https://github.com/osmosis-labs/osmosis/pull/6488#discussion_r1339189371

Didn't find a trivial way to fix and concluded that it is fine to keep as is since the change is only cosmetic